bazel/coverage/fuzz_coverage_wrapper.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+TEST_BINARY=$1
+shift
+
+mkdir -p fuzz_corpus/seed_corpus
+cp -r $@ fuzz_corpus/seed_corpus
+
+LLVM_PROFILE_FILE= ${TEST_BINARY} fuzz_corpus -max_total_time=60
```

Comment:
`-seed=<some set number>`? not sure what number, but a seed would allow make fuzzing runs determinstic